src/app/shared/services/auth/auth.service.ts

```diff
@@ -30,11 +30,11 @@ export class AuthService extends SyncServiceBase {
   }
 
   public async signInWithGoogle() {
-    return FirebaseAuthentication.signInWithGoogle();
+    return await FirebaseAuthentication.signInWithGoogle();
```

Comment:
nit(non-blocking) returning the function call already passes the promise, so adding the `await` here doesn't actually achieve anything (only useful in a try-catch statement)
https://blog.battlefy.com/when-to-use-return-vs-await-vs-return-await-vs-nothing